src/MDMTransitionContext.h

```diff
@@ -80,3 +80,9 @@ NS_SWIFT_NAME(TransitionContext)
 @property(nonatomic, strong, readonly, nonnull) UIView *containerView;
 
 @end
+
+NS_SWIFT_NAME(InteractiveTransitionContext)
+@protocol MDMInteractiveTransitionContext
```

Comment:
Ok, it might be useful to add in a flag that indicates if this is forward or backward transition